internal/tableland/tableland.go

```diff
@@ -25,7 +91,7 @@ type TxnReceipt struct {
 
 // Tableland defines the interface of Tableland.
 type Tableland interface {
-	RunReadQuery(ctx context.Context, stmt string) (interface{}, error)
+	RunReadQuery(ctx context.Context, stmt string) (*TableData, error)
```

Comment:
The core api always returns `*TableData` (previously `sqlstore.UserData`) so just making that explicit in the interface now. Also moved the definition of `TableData` to this file (it was in the store before) since it's a part of the core api now.